00-programming-fundamentals/01-how-to-osx.md

```diff
@@ -1,55 +1,150 @@
+
+
 # How to OS X: An Ode to Keyboard Shortcuts
-The mouse is slow, especially on a laptop. There are *tons* of hotkeys, tricks, and techniques, to make your usage fast and efficient entirely from the keyboard. You will certainly learn many of these during your career, we'll start by learning just a few.
-
-## Keyboard Symbols
-- `⌘` (command)
-- `⌥` (option/alt)
-- `⌃` (control)
-- `⇧` (shift)
-- `fn` (function)
-- `⇥` (tab)
-- `↩` (return/enter)
-- `␣` (space)
-
-## Hotkeys
-### Text Editing
-- `⌘s`: save current file
-- `⌘c`: copy current selection
-- `⌘x`: cut and copy current selection
-- `⌘v`: paste last copied content
-- `⌘z`: undo last action (can use multiple times)
-- `⌘⇧z`: redo last undo
-
-### Cursor & Selection
-- `⌘a`: select all contents of current window
-- `⌘l`: select current line
-- `⌘<directional arrow>`: Move cursor to maximum point in given direction
-- `⌥<left or right arrow`: Move cursor one word in either direction
-- `⇧<cursor movement command>`: select the content between start and end location of the cursor
+
+The mouse is slow, especially on a laptop. There are *tons* of hotkeys, tricks, and techniques, to make your usage fast and efficient entirely from the keyboard. You will certainly learn many of these during your career, we'll start by learning just a few.  
+
+## Learning Goals
```

Comment:
I think it'd be great to add some instructions for how to dock/snap apps to see multiple at a time. I use an app (Spectacle) but I think that OSX has it built in, right?